src/gui/activitywidget.h

```diff
@@ -160,8 +91,21 @@ public slots:
     QPushButton *_copyBtn;
 
     QSet<QString> _accountsWithoutActivities;
+    QMap<Activity::Identifier, NotificationWidget*> _widgetForNotifId;
+    QElapsedTimer _guiLogTimer;
+    QSet<int> _guiLoggedNotifications;
+    ActivityList _blacklistedActivities;
```

Comment:
Well, the "blacklisted activities" are actually these notifications that come without any action, which means that they do not display a button. These are only a few.